cypress/e2e/CMS/$CreatingAvideo/index.js

```diff
@@ -0,0 +1,76 @@
+/// <reference types="Cypress" />
+import { And } from 'cypress-cucumber-preprocessor/steps';
+function createRandomStr() {
+    var result = '';
+    var characters = 'abcdefghijklmnopqrstuvwxyz0123456789';
+    var charactersLength = characters.length;
+    for (var i = 0; i < 5; i++) {
+        result += characters.charAt(Math.floor(Math.random() * charactersLength));
+    }
+    return result;
+}
+let randomNum = createRandomStr();
+const siteSection = Cypress.env('test_site_section');
+const frontEndBaseUrl = Cypress.env('front_end_base_url');
+
+And('user fills out the following fields', (dataTable) => {
+    for (let { fieldLabel, value, field_name } of dataTable.hashes()) {
+        if (fieldLabel === 'Pretty URL') {
+            value = `${value}-${randomNum}`;
+        }
+        cy.get(`input[name^='${field_name}']`).as('inputField').parent().find('label').should('include.text', fieldLabel);
+        cy.get('@inputField').type(value);
+    }
+});
+
+And('user clicks on title with url {string} from the list of content', (contentHref) => {
+    cy.get(`a[href='${siteSection}/${contentHref}-${randomNum}']`).click();
+});
+
+And('user selects a checkbox next to title with url {string} from the list of content', (url) => {
+    cy.get(`a[href='${siteSection}/${url}-${randomNum}']`).parent().parent().find('input.form-checkbox').check();
+});
+
+Given('user is navigating to the front end site with path site section plus {string}', (purl) => {
+    cy.visit(`${frontEndBaseUrl}${siteSection}/${purl}-${randomNum}`, { retryOnStatusCodeFailure: true });
+});
+
+And('the content item with url {string} does not exist in the list of content', (url) => {
+    cy.get(`a[href='${siteSection}/${url}-${randomNum}']`).should('not.exist');
+});
+
+And('user types {string} into Caption text field', (value) => {
+    cy.getNthIframe("iframe[class='cke_wysiwyg_frame cke_reset']", 1).find('p').type(value)
+})
+
+And('user selects {string} from {string} dropdown', (dropdwon, section) => {
```

Comment:
we have this step implemented in common - why did you need to duplicate it here?

Reply:
Removed duplicated step!

Reply:
![image](https://user-images.githubusercontent.com/114245453/223270284-a5495bce-82a9-43c9-85e1-75b9b431722a.png)